scripts/tag_and_deploy

```diff
@@ -0,0 +1,164 @@
+#!/bin/bash
+
+# tag_and_deploy
+# Tags IMAGE-server components that match GitHub Actions workflows with tag triggers
+# Usage:
+#	./scripts/tag_and_deploy # tags all services
+#	./scripts/tag_and_deploy services/tat # tags 1+ services
```

Comment:
This means that it can take more than one argument for explicit services, correct?